src/main/resources/templates/personListPage.html

```diff
@@ -6,6 +6,7 @@
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
 </head>
 <body>
+
```

Comment:
I don't think any of the changes for this file need to stay

Reply:
What do you mean?

Reply:
Like the changes were just adding the whitespace

Reply:
yeah it sometimes does that